programs/bpf_loader/src/lib.rs

```diff
@@ -146,6 +146,9 @@ pub fn create_executor(
         reject_callx_r10: invoke_context
             .feature_set
             .is_active(&reject_callx_r10::id()),
+        dynamic_stack_frames: false,
+        enable_sdiv: false,
+        optimize_rodata: false,
```

Comment:
We can add features for these items in later PRs, @alessandrod I think these are all yours correct?

Reply:
Correct! Yup sure there's no rush. For the first two we don't have a (released) toolchain that can exercise those features yet anyway, and for optimize_rodata I want to land https://github.com/solana-labs/rbpf/pull/301 first